docker/docker-dev-debug/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '2'
+
+services:
+  metrictank:
+    security_opt:
+      - seccomp:unconfined
+    hostname: metrictank
+    image: grafana/metrictank:latest-debug
+    ports:
+     - "6060:6060"
+     - "2003:2003"
+     - "40000:40000"
+    volumes:
+      - ../../build/metrictank:/usr/bin/metrictank
+      - ./metrictank.ini:/etc/metrictank/metrictank.ini
+      - ./storage-schemas.conf:/etc/metrictank/storage-schemas.conf
```

Comment:
do we need the custom storage-schemas.conf and storage-aggregation.conf?

Reply:
I suppose we don't need custom storage-schemas or aggregation configuration files. I will remove them.